coalib/bearlib/abstractions/Lint.py

```diff
@@ -43,7 +43,7 @@ class Lint(Bear):
     Deals with the creation of linting bears.
 
     For the tutorial see:
-    http://coala.readthedocs.org/en/latest/Users/Tutorials/Linter_Bears.html
+    https://coala.readthedocs.io/en/stable/Users/Tutorials/Linter_Bears.html
```

Comment:
rather use docs.coala.io instead of coala.readthedocs.io, as it's shorter (does the same)